src/components/AccountPicker.jsx

```diff
@@ -0,0 +1,88 @@
+import styles from '../styles/accountPicker'
+import classNames from 'classnames'
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { translate } from 'cozy-ui/react/i18n'
+
+import { getConnectionsByKonnector } from '../reducers'
+import { getRegistryKonnector } from '../ducks/registry'
+
+import Icon from 'cozy-ui/react/Icon'
+import Modal, { ModalHeader, ModalContent } from 'cozy-ui/react/modal'
```

Comment:
There is a typo here, it's `cozy-ui/react/Modal` since React components begin by an uppercase letter.
This is currently breaking Travis build.